test/AngularSpec.js

```diff
@@ -435,6 +435,95 @@ describe('angular', function() {
   });
 
 
+  describe('jq', function() {
+    var element;
+
+    beforeEach(function() {
+      element = document.createElement('html');
+    });
+
+    afterEach(function() {
+      jq.name_ = undefined;
+      delete jq.name_;
+    });
+
+    it('should return undefined when jq is not set, no jquery found (the default)', function() {
+      expect(jq()).toBe(undefined);
+    });
+
+    it('should return empty string when jq is enabled manually via [ng-jq] with empty string', function() {
```

Comment:
@gkalpak @mzgol Again, open to suggestions.  This is my first time coding angular itself and I'm not sure what would be the best approach.